src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. Error 5: Inconsistent naming convention
   - The method `Subtract` does not follow the standard naming convention as it starts with an uppercase letter.